documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -594,7 +594,7 @@ For further examples, there is a large collection of overlay source files https:
 
 The overlay handling in the firmware and the run-time overlay application using the `dtoverlay` utility treat labels defined in an overlay as being private to that overlay. This avoids the need to invent globally unique names for labels (which keeps them short), and it allows the same overlay to be used multiple times without clashing (provided some tricks are used - see <<part2.2.9,Special properties>>).
 
-Sometimes, however, it is very useful to be able to create a label with one overlay and use it from another. Firmware released since 14th February 2020 has the ability to declare some labels as being global - the `__export__` node:
+Sometimes, however, it is very useful to be able to create a label with one overlay and use it from another. Firmware released since 14th February 2020 has the ability to declare some labels as being global - the `+__exports__+` node:
```

Comment:
@pelwell I changed this to `__exports__` because that's what's used in the code-block immediately below?

Reply:
That's correct - it should be plural like the other similar nodes (__symbols__, __fixups__, etc.).